inspirehep/modules/records/jsonschemas/records/elements/id.json

```diff
@@ -0,0 +1,155 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "anyOf": [
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE ID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "INSPIRE-\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "INSPIRE BAI"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "(\\w+\\.)+\\d+",
+                    "type": "string"
+                }
+            },
+            "type": "object"
+        },
+        {
+            "properties": {
+                "type": {
+                    "enum": [
+                        "ORCID"
+                    ],
+                    "type": "string"
+                },
+                "value": {
+                    "pattern": "\\d{4}-\\d{4}-\\d{4}-\\d{3}[0-9X]",
```

Comment:
In fact would be great to rather normalize all these IDs with before_record_update/insert enhancer :dancers: